.travis.yml

```diff
@@ -6,6 +6,11 @@ rust:
 
 cache: cargo
 
+matrix:
+  fast_finish: true
+  allow_failures:
+    - rust: nightly
```

Comment:
To work around failures due to nightly not having rustfmt installed. Alternatively, manually install nightly with the required components specified so rustup will look for the latest nightly that has rustfmt. I don't think travis has an easy way to do this.

Reply:
Realizing that the example was perhaps deliberately simplified to make it buildable on CI.

Reply:
I think it was simple from the beginning :)